test/games/am2r/generator/test_am2r_base_patches.py

```diff
@@ -4,54 +4,94 @@
 import uuid
 from random import Random
 
+import pytest
+
+from randovania.game_description.db.dock import DockWeakness
 from randovania.games.am2r.generator.base_patches_factory import AM2RBasePatchesFactory
 from randovania.games.am2r.layout.am2r_configuration import AM2RConfiguration
 from randovania.layout import filtered_database
 
-_door_list = [
-    1047,
-    1751,
-    1259,
-    596,
-    1258,
-    1279,
-    1280,
-    1454,
-    1137,
-    878,
-    409,
-    1246,
-    1248,
-    1330,
-    1281,
-    1336,
-    1331,
-    1393,
-    1361,
-    1491,
-    1257,
-    1416,
-    1614,
-    1561,
-    1720,
-    1656,
-    601,
-]
-
-
-def test_base_patches(am2r_game_description, preset_manager) -> None:
+_save_door_mapping = {
+    37: "Normal Door",
+    41: "Normal Door",
+    305: "Normal Door",
+    342: "Normal Door",
+    495: "Normal Door",
+    498: "Normal Door",
+    1051: "Normal Door",
+    1057: "Normal Door",
+    1058: "Normal Door",
+    1062: "Normal Door",
+    1185: "Normal Door",
+    1190: "Normal Door",
+    1191: "Normal Door",
+    1194: "Normal Door",
+    1277: "Normal Door",
+    1302: "Normal Door",
+    1303: "Normal Door",
+    1306: "Normal Door",
+    1339: "Normal Door",
+    1343: "Normal Door",
+    1345: "Normal Door",
+    1348: "Normal Door",
+    1736: "Normal Door",
+    1740: "Normal Door",
+    1741: "Normal Door",
+    1745: "Normal Door",
+    1764: "Normal Door",
+    1769: "Normal Door",
+    1770: "Normal Door",
+    1775: "Normal Door",
+    1776: "Normal Door",
+    1779: "Normal Door",
+    1780: "Normal Door",
+    1787: "Normal Door",
+    1788: "Normal Door",
+    1796: "Normal Door",
+    1797: "Normal Door",
+    1804: "Normal Door",
+    1805: "Normal Door",
+    1808: "Normal Door",
+    1889: "Normal Door",
+    1894: "Normal Door",
+}
+
+_lab_door_mapping: dict = {}
```

Comment:
This thing seems to be only referenced one time, which is when iterating over it in Line 94.